2025/selection-process.md

```diff
@@ -0,0 +1,183 @@
+
+As in previous years proposals can either be for focus areas or
+investigations. For simplicity the remainder of this document will
+only refer to focus area proposals, but this also covers proposals for
+investigation efforts.
+
+### Champion Selection
+
+**Timeline**: October 3rd, 10th, 17th
+
+Interop participants propose focus areas, consisting of one or more
+proposals, that they would like to champion. Proposals will initially
+be shared in time for the meeting on October 10th, and champions
+finalised on October 17th.
+
+Where more than one participant wants to champion the same focus area
+proposal a single champion must be nominated for the remainder of the
+process.
+
+Champions and seconds (i.e. organizations that would champion a focus
+area but were not the final selected champion) are recorded for each
+proposal.
+
+There is no assigned limit to the number of proposals one participant
+may champion.
+
+_Alternative_: Each participant is expected to champion a minimum of
+three proposals.
+
+In case different focus areas are put forward with overlapping sets of
+proposals, the participants are encouraged to work out a way to divide
+the proposals into focus areas in a way that satisfies everyone. If
+this is impossible then carrying forward multiple competing focus area
+proposals is permitted, but final selections will only allow each
+proposal to appear in a single focus area.
+
+Any proposal without a champion will not be carried forward.
+
+### Focus Area Proposal Refinement
+
+**Timeline**: October 24th and 31st
+
+Champions gather evidence of the impact of their proposals, using the
+rubric developed earlier. Specific participants may also be asked
+to provide (publicly available) information for all proposals where
+that participant has ownership of the information (e.g. standards
+positions, use counter data).
+
+### Focus Area Proposal Presentation
+
+**Timeline**: November 7th and 14th
+
+Each participant gets 15 minutes of meeting time to present the
+proposals they are championing, and make the case that they should be
+considered a priority for inclusion in Interop 2025.
+
+_Alternative_: Have a longer meeting on either November 7th or November
+14th so that all the proposals are presented in the same week.
+
+### Focus Area Proposal Ranking
+
+**Timeline**: Before December 5th
+
+Participants rank each proposal as P1 (highest priority) to P3 (lowest
+priority), or veto.
+
+There is no preset limit on how many proposals each participant may
+assign each rank.
+
+Initial rankings are submitted in time for the December 5th meeting.
+
+### Focus Area Ranking and Selection
+
+**Timeline**: December 5th, 12th, 19th
+
+Focus areas with any vetoes are eliminated.
+
+Proposals that have strong positive consensus (majority P1 rankings,
+no P3 rankings) are immediately adopted.  Proposals with a strong
+negative consensus (majority P3 rankings, no P1 ranking, or P1 only
+from the champion) are immediately dropped.
```

Comment:
Does majority here mean 4 given that we have 6 participating organizations? The details here might change the number of proposals that are covered by this by a lot.

Reply:
I've changed the wording to make it clear that it remains a judgement call, not an algorithm. Given that it's a new process it seems better to allow additional flexibility here.